### PR TITLE
feat added ui

### PR DESCRIPTION
**Pull request by deepak29-git**
1.add ui
2.create image url

### CODE REVIEW

File: src/Components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+import { Modal2 } from "../Modal2/Modal2";
```
Comment (bajajcodes) [inline src/Components/Button/Button.jsx:2]: Change name to something meaningful and self-describing, `Modal2` does not make sense.

File: src/Components/Button/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import "../Button/Button.css";
+export const Button = () => {
+  const [show, setShow] = useState(false);
+  return (
+    <>
+      <div className="button-container">
+        <button onClick={() => (!show ? setShow(true) : setShow(false))}>
```
Comment (bajajcodes) [inline src/Components/Button/Button.jsx:9]: We don not need a button to show modal here even if we make a web app.
Hence, modal can be displayed or made visible directly on main app either it is web app or web extension.

File: src/Components/Modal/Modal.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import "../Modal/Modal.css";
+
+export const Modal = () => {
+  const [image, setImage] = useState("");
+  const onImageChange = (e) => {
```
Comment (bajajcodes) [inline src/Components/Modal/Modal.jsx:6]: This piece of code, I guess needs to be removed.

**Review by bajajcodes — CHANGES_REQUESTED** [review]
- Changes are suggested in comments.
- Refer tweet2Notion for design.

**Review by arpit01923 — COMMENTED** [review]
- looks good
- one suggestion, improve naming convention

File: src/Components/Button/Button.jsx (hunk 1 above)
Comment (deepak29-git) [reply src/Components/Button/Button.jsx:2]: done

**Pull request merged**